run_dir/static/js/running_notes.js

```diff
@@ -147,20 +148,66 @@ function filter_running_notes(search){
         }
     });
 }
+//Count rn's of each category and add badge with number to filter dropdown
+function count_cards(){
+    var cat_cards = {};
+    var all = 0;
+    $('#running_notes_panels').find('.badge').each(function(){
+        var label = $.trim($(this).text());
+        all++
+        if (label){
+            if (label in cat_cards){
+                cat_cards[label]++;
+            }else{
+                cat_cards[label] = 1;
+            }
+        }
+    });
+    $('.btn_count').append('All'+'&nbsp;'+'<span class="badge bg-secondary">'+all+'</span>');
+    $('#rn_category').next().find('.dropdown-item').each(function(){
+        var label = $.trim($(this).text())
+        if (label == 'All'){
+            $('.all_count').prepend('<span class="badge bg-secondary">'+all+'</span>'+'&nbsp;');
+        }else if (!cat_cards[label]){
+            $(this).parent('li').addClass('d-none');
+        }else{
+            if (label == 'Workset'){
```

Comment:
This if-else if ladder could probably be cut down with a dictionary with the Categories as keys and the `bg-*` as values, would be easier to update too.

Reply:
True, good suggestion!